pkg/packer/manifests/gcp/packer.json.tmpl

```diff
@@ -0,0 +1,203 @@
+{
+  "variables": {
+    "account_file": "{{ env `GOOGLE_APPLICATION_CREDENTIALS` }}",
+    "ansible_extra_vars": "{{ user `ansible_extra_vars` }}",
+    "build_name": "",
+    "build_name_extra": "",
+    "build_timestamp": "{{ timestamp }}",
+    "gpu": "false",
+    "gpu_nvidia_version": "",
+    "gpu_types": "",
+    "konvoy_image_builder_version": "0.0.1",
+    "kubernetes_full_version": "",
+    "manifest_output": "manifest.json",
+    "network": "",
+    "project_id": "",
+    "zone": ""
+  },
+  "builders": [
+    {
+       "image_name": "{{ (user `distribution`) | lower }}-{{ user `distribution_version` }}{{ user `build_name_extra` }}",
+       "image_family": "{{ user `distribution_family` }}",
+       "type": "googlecompute",
+       "project_id": "{{ user `project_id` }}",
+       "source_image": "{{ user `source_image` }}",
+       "source_image_family": "{{ user `distribution_family` }}",
+       "zone": "{{ user `zone` }}",
+       "ssh_username": "packer",
+       "ssh_key_exchange_algorithms": [
+         "curve25519-sha256@libssh.org",
+         "ecdh-sha2-nistp256",
+         "ecdh-sha2-nistp384",
+         "ecdh-sha2-nistp521",
+         "diffie-hellman-group14-sha1",
+         "diffie-hellman-group1-sha1"
+        ],
+       "wait_to_add_ssh_keys": "20s",
+       "image_labels": {
+         "build_date": "{{ isotime | clean_resource_name }}",
+         "build_timestamp": "{{ user `build_timestamp` | clean_resource_name }}",
+         "containerd_version": "{{ user `containerd_version` }}",
+         "gpu": "{{user `gpu`}}",
+         "gpu_nvidia_version": "{{ user `gpu_nvidia_version` }}",
+         "gpu_types": "{{ user `gpu_types` }}",
+         "image_builder_version": "{{ user `konvoy_image_builder_version` | clean_resource_name }}",
+         "kubernetes_cni_version": "{{ user `kubernetes_cni_version` }}",
+         "kubernetes_version": "{{ user `kubernetes_full_version` | clean_resource_name }}"
+       },
+       "metadata": {
+
+       },
+       "network": "{{ user `network` }}",
+       "tags": [
+         "alejandro-network-allow-ssh"
```

Comment:
please remove hardcoded tag and use variables.

Reply:
oh thank you for pointing that out!